### ISSUE TITLE
New single host, multi-node tutorial

### ISSUE THREAD

**Issue by gleehokie** [EOSIO/eos#2168]
On a single host:

- Start a local node
- Start a 2nd node that talks to the 1st (master-slave)
- Use the BIOS contracts to start 2 producers
- Avoid having instructions that involve changing the config files, use the API

To replace the tutorial on the Wiki

**Comment by pmesnier** [comment]
PR submitted to the private tutorial repo. The four steps enumerated above are featured, along with the intermediate steps necessary to get from one to the next.

**Comment by gleehokie** [comment]
@rbw3821 please review this and integrate it with the tutorials in the Wiki

**Comment by pmesnier** [comment]
@rbw3821  are you reviewing my PR? I don't mention config file edits.

**Comment by rbw3821** [comment]
Yep, after reviewing I see that there are no references to config file edits.  Just doing some minor cleanup and will merge in a few.

**Comment by pmesnier** [comment]
Rick, there is a fair amount of overlap between this and the contract tutorial. Perhaps you could factor out the common part into a separate "getting started" tutorial that is referenced by both?

**Comment by rbw3821** [comment]
I've tested and it has been tested by QA.  Will merge to eos.wiki on Thursday.
